docs/lesson/improvements.rst

```diff
@@ -0,0 +1,301 @@
+2. Click |sendandclose| when done.
+
+.. caution:: 
+   Leave the status at *Open* if you are still working on the task. Only use the other options if you want to hide the task from the Improvements screen.
+
+.. |reviewthread| image:: /images/review_thread_button.png
+                  :alt: Review Thread button
+                  :scale: 40 %
+
+.. figure:: /images/review_thread.png
+   :alt: Review feedback message box
+   :scale: 40 %
+
+   *Fig. 3*
+
+.. _suggestions:
+
+Suggestions for a card
+-----------------------
+
+Learners also have the option to suggest an *edit* on one or more cards, e.g., a change in the content, formatting etc. This comes through as a *Suggestion* task in the Improvements screen.
+
+.. figure:: /images/suggestion_task.png
+   :alt: Suggestion task
+   :scale: 40 %
+
+   *Fig. 4*
+
+1. Click the |reviewthread| button and then the |reviewsuggestion| button to take action on this suggestion.
+
+.. |reviewsuggestion| image:: /images/review_suggestion_button.png
+                      :alt: Review Suggestion button
+                      :scale: 35 %
+
+2. After reviewing the changes suggested by the learner, you can either accept or reject the suggestion.
+
+.. figure:: /images/review_suggestion.png
+   :alt: Review suggestion box
+   :scale: 40 % 
+
+   *Fig. 5*
+
+3. Enter a new message if you wish to add to the message thread and change the status of the suggestion as required. Finally, click the |sendandclose| button. 
+
+.. |sendandclose| image:: /images/send_and_close.png
+                  :alt: Send and Close button
+                  :scale: 40 %
+
+.. figure:: /images/suggestion_message.png
+   :alt: Suggestion message box
+   :scale: 30 %
+
+   *Fig. 6*
+
+.. _answerdetails:
+
+Answer details for a card
+--------------------------
+
+If you checkmarked the **Solicit Answer Details** box while creating your exploration, Oppia will occasionally ask a learner to explain why they chose a certain answer for that card.
+
+.. figure:: /images/solicit_answer_details.png
+   :alt: Solicit answer details checkbox
+   :scale: 40 %
+
+   *Fig. 7*
+
+Whenever a learner explains why they chose their answer for a card, you will see the details in the Improvements screen as shown below.
+
+.. figure:: /images/answer_details_task.png
+   :alt: Answer details task
+   :scale: 40 %
+
+   *Fig. 8*
+
+1. To review the details, click the |reviewanswerdetails| button and click under **Answer details** to expand the comment.
+
+.. |reviewanswerdetails| image:: /images/review_answer_details.png
+                         :alt: Review Answer Details box
+                         :scale: 40 %
+
+.. figure:: /images/answer_details_review.png
+   :alt: Review answer details
+   :scale: 40 %
+
+   *Fig. 9*
+
+
+2. To delete the **Answer details** task, checkmark the **Select** box and click on the |deleteitems| box.
```

Comment:
To delete any **Answer details** received, checkmark the **Select** box and click on......
You can also delete multiple answer details at a time.

Reply:
Done